### ISSUE TITLE
Toolbar configuration: the "headings" "alignment" and "styles" buttons cannot be focused via keyboard navigation

### ISSUE THREAD

**Issue by klonos** [backdrop-contrib/ckeditor5#35]
The reason for this is that these are `<span>`s, while the rest of the toolbar buttons are `<a>`s.

We should try to see if setting a `tabindex` attribute solves that problem, or alternatively change these to `<a>` elements as well (only for the toolbar configuration form).

**Comment by klonos** [comment]
> We should try to see if setting a `tabindex` attribute solves that problem...

That should work. I just checked the respective HTML in Drupal (where tabbing works), and they seem to be adding `tabindex="0"` to all `<li>`s (along with a `role="option"` attribute btw).

**Comment by quicksketch** [comment]
Using `tabindex="0"` works great. I focusing these special buttons as part of the PR at https://github.com/backdrop-contrib/ckeditor5/pull/92.
